### PR TITLE
Added replace line to escape unicode characters

### PR DESCRIPTION
**Pull request by appienvanveen**
Added *.iml to .gitignore

## I'm submitting a...

```
[x ] Bug Fix
[ ] Feature
[x ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- other: extended gitignore
- fix: 
https://github.com/dherges/ng-packagr/issues/425

## Description
See: 
https://github.com/dherges/ng-packagr/issues/425

## Does this PR introduce a breaking change?

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by appienvanveen** [comment]
Hi @dherges,

can you please have a look a this PR? We are using your great tool at our enterprise and looking forward to this fix. If this solution does not suits you, please point the right direction.

**Comment by dherges** [comment]
Hi @appienvanveen,

sorry for my late reply.

Can you add a test that reproduces the original issue and verifies the new correct behaviour?

In the issue, you mentioned a css stylesheet like:

```css
.foo {
  content: "\2014 \00A0"; // em dash, nbsp
}
```

Can you add a component (plus the css file) to one of the `integration/samples/*` libs?

The test should then verify that the component's style contains the correct css rule – currently, the most reliable way to do so is through the metadata json. The test should `JSON.parse(..)` the metadata json file, locate the component, extract the component's style attribute and verify like `expect(stylesheetString).to.contain('/* what you expect here *)'` – there are already test specs like https://github.com/dherges/ng-packagr/blob/master/integration/samples/material/specs/metadata.ts#L38-L40 and you can do similar

**Comment by dherges** [comment]
Code example listing:

https://github.com/dherges/ng-packagr/blob/17b4e0f/integration/samples/material/specs/metadata.ts#L38-L40

Does it work??!?

**Comment by appienvanveen** [comment]
Hi @dherges! Thanks for replying! 

I've extended the baz integrations test with the unicode parsing test. Hopefully I did what you meant. 
Please let me know if it is okay. If not, I will update the code according to your comments.

File: integration/samples/material/baz/baz.component.scss [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 .baz {
     color: $variable-defined-by-file-not-processed-by-loader;
+    content: "\2014 \00A0";
```
Comment (dherges) [inline integration/samples/material/baz/baz.component.scss]: Isn't that supposed to be valid unicode character in the css?

https://stackoverflow.com/a/10393517

Comment (dherges) [reply integration/samples/material/baz/baz.component.scss]: Ok, got it. The unicode character triggers `SyntaxError: Octal literal in strict mode`. Not so good....

**Comment by dherges** [comment]
For clarification / memo to self, see https://brajeshwar.github.io/entities/

Related to https://github.com/styled-components/styled-components/issues/188

**Comment by appienvanveen** [comment]
When a new (beta) release will be made including this fix?

**Pull request merged**